.env.example

```diff
@@ -0,0 +1,5 @@
+APP_NAME='Sage'
+APP_ENV='development'
+APP_SSL=false
+APP_HOST='example.test'
+APP_PUBLIC_PATH='app/themes/sage/dist'
```

Comment:
I am a bit hesitant with the naming of this file since it could give the impression that secrets go in here which could be devastating for since the theme is public.

Reply:
Do you think it would be sufficient to simply add a warning at the top?

We could be obnoxiously loud about it:

```
# ⚠ ⚠ ⚠ DO NOT STORE SECRETS HERE!
```

Reply:
Ya, I thought about that too. It could work.

Reply:
This is a little more complex and a lot more flexible since v2 launch.

Current behavior:

  - Bud only includes env values prefixed by `APP_` in bundled code
  - Bud will explicitly ignore env values including `SECRET` in their property name

Examples:

- `APP_NAME` -- included in bundle
- `APP_SECRET_KEY` -- not included in bundle
- `SOME_UNRELATED_KEY` -- not included in bundle